wiki/en/Running-a-Server.md

```diff
@@ -286,6 +286,12 @@ You can see the result of these commands if you run `systemctl status jamulus`,
 
 ---
 
+## Adding metadata to the Server
+
+You can add metadata to the welcome message of a Server to add additional, hidden information like contact information or policy requests for Bots (this is comparable to [robots.txt](https://en.wikipedia.org/wiki/Robots.txt)). See the [Community Knowledge Base entry about metadata](/kb/2023/07/30/Server-Metadata.html) for more information.
```

Comment:
```suggestion
You can add metadata to the welcome message of a Server to add additional, hidden information like contact information or policy requests for bots (this is comparable to [robots.txt](https://en.wikipedia.org/wiki/Robots.txt)). See the [Community Knowledge Base entry about metadata](/kb/2023/07/30/Server-Metadata.html) for more information.
```

Capitalization is a Whole Area. I recommend limiting capitalization to products we release.